dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -287,7 +219,27 @@
       isCleverStudent: current_user.clever_student?,
       dependentStudentsCount: Queries::User.dependent_students_count(current_user.id),
       personalAccountLinkingEnabled: @personal_account_linking_enabled,
-      lmsName: Queries::Lti.get_lms_name_from_user(current_user)
+      lmsName: Queries::Lti.get_lms_name_from_user(current_user),
+      authorizedTeacher: current_user.permission?(UserPermission::AUTHORIZED_TEACHER),
+      canChangeUserType: current_user.can_change_own_user_type?,
+      secretPictureAccountOnly: current_user.secret_picture_account_only?,
+      errors: current_user.errors,
+      migrated: current_user.migrated?,
+      parentManagedAccount: current_user.parent_managed_account?,
+      teacherManagedAccount: current_user.teacher_managed_account?,
+      isStudent: current_user.student?,
+      shouldSeeEditEmailLink: current_user.should_see_edit_email_link?,
+      hashedEmail: current_user.hashed_email,
+      encryptedPasswordPresent: current_user.encrypted_password.present?,
+      canEditPassword: current_user.can_edit_password?,
+      sponsored: current_user.sponsored?,
+      showGenderInput: experiment_value('gender', request),
+      userTypeOptions: user_type_options,
+      usStateDropdownOptions: us_state_options,
```

Comment:
@dmcavoy Should we create a small ticket to create a similar state options constant to the `User::AGE_DROPDOWN_OPTIONS` below? Since we're doing a lot with signup and account stuff it seems like it would be a good constant to have (unless if in certain cases we don't show a certain state?)

Reply:
I'm not sure I'm the best person to decide on this. @bethanyaconnor may have been inputs here

Reply:
I think we do have a constant list of states somewhere, no? In https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/geographyConstants.js?

Reply:
oh, I think this merged already with the AccountInformation pr. I branched off of that branch to create this pr.

I passed those values from backend to front through scriptData before I knew about the sharedConstants generated file, but a refactor does make sense. maybe a clean up ticket?